.github/workflows/test_tutorial_workflow.yml

```
@@ -19,13 +19,16 @@ concurrency:
 
 jobs:
   test:
+    strategy:
+      matrix:
+        python-version: ['3.7', '3.11']
```

I've tested the python version after setup-miniconda has run, it's showing the same version as setup-python. So this should be fine:
```suggestion
        python-version: ['3.7', '3']
```

https://github.com/MetRonnie/cylc-flow/actions/runs/4883788233/jobs/8715616835 (not sure why it failed there)